ui/settings/mainContent.css

```diff
@@ -25,7 +25,7 @@ QMessageBox
 
 QCheckBox
 {
-    background: white;
+    background: #FCFCFC;
```

Comment:
I agree, but if I remove this rule, I have this
![spectacle lh1806](https://cloud.githubusercontent.com/assets/6306608/16711924/44dd9738-4685-11e6-85d3-3049056e2b61.png)